.quickstart/quickstart.yml

```diff
@@ -18,3 +18,13 @@ destination_configurations:
     dispatch:
       - macro_namespace: dbt_utils
         search_order: [ 'spark_utils', 'dbt_utils' ]
+
+public_models: [
+  "zuora__billing_history",
+  "zuora__account_daily_overview",
+  "zuora__line_item_history",
+  "zuora__subscription_overview",
+  "zuora__monthly_recurring_revenue",
+  "zuora__account_overview",
+  "zuora__line_item_enhanced"
```

Comment:
@fivetran-joemarkiewicz this model is [disabled](https://github.com/fivetran/dbt_zuora/tree/MagicBot/documentation-updates?tab=readme-ov-file#enabling-standardized-billing-model) by default right? Does that mean we shouldn't include it here since Quickstart folks can't turn it on?

Reply:
If so, we'll need to adjust the other READMEs of the the other packages included in the standardized billing report (and any other disabled-by-default models)

Reply:
will address in future ticket